benchmarks/user/README.md

```diff
@@ -211,6 +211,51 @@ Do not forget to pin processes, especially on NUMA systems; below, we do so with
 ```
 numactl --cpubind=0 --membind=0 python benchmark.py ...
 ```
+
+## Benchmarking performance
+
+In order for a user to benchmark the performance of an operator calculating
```

Comment:
This paragraph is useless since the last bench-related PR landed. See [this line](https://github.com/opesci/devito/blob/master/benchmarks/user/benchmark.py#L397) and [this line](https://github.com/opesci/devito/blob/master/benchmarks/user/benchmark.py#L405)

Reply:
you may replace it with something along the lines of:

_The GFlops/s and GPoints/s performance, Operational Intensity (OI) and execution time are emitted to standard output at the end of each run. Further, when running in ``bench`` mode, a \`.json\` file is produced (see \`python benchmark.py bench --help\` for more info)